src/OrchardCore.Modules/OrchardCore.Media/Controllers/MediaCacheController.cs

```diff
@@ -48,6 +56,13 @@ public async Task<IActionResult> Purge()
                 return Unauthorized();
             }
 
+
+            if (_mediaFileStoreCache == null)
+            {
+                _notifier.Error(H["The asset cache feature is enabled, but a remote media store feature is not enabled, or not configured with appsettings.json."]);
```

Comment:
Is this an error or could be a warning?!!

Reply:
Nope, it's an error